conda_env/installers/base.py

```diff
@@ -1,6 +1,4 @@
-import pkg_resources
-
-ENTRY_POINT = 'conda.env.installers'
+ENTRY_POINT = 'conda_env.installers'
```

Comment:
Nope -- it was not a type -- the entry point was based on what the eventual name was supposed to be.  The idea is that eventually `conda_env` would become `conda.env`.